corehq/messaging/scheduling/forms.py

```diff
@@ -1344,6 +1347,24 @@ class ScheduleForm(Form):
         required=False,
     )
 
+    use_user_case_for_filter_choices = [(NO, "User Properties"), (YES, "User Case Properties")]
+
+    use_user_case_for_filter = ChoiceField(
+        label=gettext_lazy("Filter on"),
+        required=False,
+        initial=NO,
+        choices=use_user_case_for_filter_choices,
+        widget=SelectToggle(
```

Comment:
I'm not sure where this is controlled, but this should be disabled when viewing an alert that's already created. It's currently still enabled

Reply:
Why? You can change what recipients to send to on edit. I think being able to also edit the filter would makes sense.

Reply:
I should have specified  I was referring to specifically an immediate broadcast. After you create an "immediate" broadcast, you can view a read-only version of the form. I agree with you that for the other views that can be edited, those should continue being editable.

Reply:
https://github.com/dimagi/commcare-hq/pull/35336